# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gdm/minimal_uid.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Users are >= 500, not 100, as defined in /etc/login.defs.
This is useful when you enable 'browse' on gdm.
Highest system UID I know is rsbac, 400.
Alejandro, jsaw

--- gdm-2.8.0.7/config/gdm.conf.in	2005-10-06 21:25:45.000000000 +0200
+++ gdm-2.8.0.7-t2/config/gdm.conf.in	2006-01-29 01:54:36.000000000 +0100
@@ -282,7 +282,7 @@
 # face browser or in the gdmselection list for Automatic/Timed login.
 # They will not be displayed regardless of the settings for
 # Include and Exclude.
-#MinimalUID=100
+MinimalUID=500
 # Users listed in Include will be included in the face browser and in
 # the gdmsetup selection list for Automatic/Timed login.  Users
 # should be separated by commas.
